test/functional/testlib.bash

```diff
@@ -1986,10 +2007,40 @@ update_bundle() {
 
 	# re-order items on the manifest so they are in the correct order based on version
 	sudo sort -t$'\t' -k3 -s -h -o "$bundle_manifest" "$bundle_manifest"
-
 	update_manifest -p "$bundle_manifest" contentsize "$contentsize"
 	update_manifest -p "$bundle_manifest" timestamp "$(date +"%s")"
 
+	if [ "$iterative" = true ]; then
+		# create the iterative "to manifest"
+		to_manifest="$bundle_manifest".I."$oldversion"
+		# get the manifest header as is
+		to_manifest_content=$(sed "/^[^\\t]*\\t[^\\t]*$/! d" "$bundle_manifest")$'\n'$'\n'
+		# get only those bundles that match the latest version
+		to_manifest_content+=$(sed "/^....\\t.*\\t$version\\t.*$/! d" "$bundle_manifest")$'\n'
+		write_to_protected_file "$to_manifest" "$to_manifest_content"
+		update_manifest "$to_manifest" filecount "$(grep -c -E "^(F|D)..." "$to_manifest")"
```

Comment:
Are directories out of scope for iterative manifests, since deltas are not applied for them? If so, the regex should be `^F...` instead. Same comment applies for the others spots that use this regex.

Reply:
that is a good question, probably directories are out of scope as you mention

Reply:
No, directories are out of scope for the delta from manifest, not for the iterative to manifest. The iterative to manifest has everything that changed in the latest release, including links and directories. So you need to include `L` in this regex.

Reply:
I'll add the Links to the pattern